src/components/typography/typography.stories.mdx

```diff
@@ -126,6 +126,79 @@ It's also possible to create ordered and unordered lists using the `List`, `List
   </Story>
 </Preview>
 
+## Examples of white-space
+The `whiteSpace` prop sets how white space inside of an element is handled. The `whiteSpace` prop accepts any valid 
+CSS string.
+
+<Preview>
+  <Story name="white-space" parameters={{ info: { disable: true } }}>
+    {() =>
+      ["normal", "nowrap", "pre", "pre-wrap", "pre-line", "break-spaces"].map((variant) => (
+        <div style={{padding: "32px"}}>
+          <div style={{height: "150px", width: "150px", backgroundColor: "yellow"}}>
+            <Typography whiteSpace={variant}>
+              The is an example of using white-space. This is an example of some text with {variant} applied.
+            </Typography>
+          </div>
+        </div>
+      ))
+    }
+  </Story>
+</Preview>
+
+## Examples of word-wrap
+The `wordWrap` prop allows longer words to be broken and wrapped on to the next line. The `wordWrap` prop
+accepts any valid CSS string.
+
+<Preview>
+  <Story name="word-wrap" parameters={{ info: { disable: true } }}>
+    {() =>
+      ["normal", "break-word", "initial", "inherit"].map((variant) => (
+        <div style={{padding: "32px"}}>
+          <div style={{height: "100px", width: "150px", backgroundColor: "yellow"}}>
+            <Typography wordWrap={variant}>
+              The is an example of using word-wrap. This is an example of some text with {variant} applied.
+            </Typography>
+          </div>
+        </div>
+      ))
+    }
+  </Story>
+</Preview>
+
+## Examples of text-overflow
+The `textOverflow` prop specifies how overflowed content that is not displayed should be signaled to the user. 
+The `textOverflow` prop accepts any valid CSS string.
+Please note that the element must have `overflow="hidden"` and `whiteSpace="nowrap"` set for `textOverflow` to work correctly.
+
+<Preview>
+  <Story name="text-overflow" parameters={{ info: { disable: true } }}>
+    {() =>
+      ["clip", "ellipsis", "string", "initial", "inherit"].map((variant) => (
+        <div style={{padding: "32px"}}>
+          <div style={{height: "100px", width: "150px", backgroundColor: "yellow"}}>
+            <Typography textOverflow={variant} whiteSpace="nowrap" overflow="hidden">
+              The is an example of using textOverflow. This is an example of some text with {variant} applied.
+            </Typography>
+          </div>
+        </div>
+      ))
+    }
+  </Story>
+</Preview>
+
+## Example of truncate
+The `truncate` prop is a boolean prop that automatically applies the styles required to trunacte a piece of text.
```

Comment:
I'd leave this one though because it's not a direct `propName => css rule` situation (the ones shown above are) but it's something custom